examples/model_selection/plot_grid_search_refit_callable.py

```diff
@@ -79,49 +101,293 @@ def best_low_complexity(cv_results):
     return best_idx
 
 
+# %%
+# Set up the pipeline and parameter grid
+# -------------------------------------
+#
+# We create a pipeline with two steps:
+# 1. Dimensionality reduction using PCA
+# 2. Classification using LinearSVC
```

Comment:
I think I would always recommend using `LogisticRegression` over linear SVC nowadays. Those models have similar ROC-AUC capabilities, but only LR can output interpretable confidence scores with `predict_proba` (and evaluated with a proper scoring rule such as Brier score or log loss).

Furthermore, Liblinear `sample_weight` support [seems to be broken in subtle ways](https://github.com/scikit-learn/scikit-learn/issues/16298) that might be difficult to fix, so I would rather stop implicitly recommending this model in our examples.